src/renderer/missionWindow/MissionWindow.tsx

```diff
@@ -485,11 +547,15 @@ export default class MissionWindow extends Component<ThemeProps, State> {
      * (and no mission is running).
      */
 
-    const unlockStartMissionButton = information[missionName].parameters !== undefined
-      && checklist.geofenceTop !== undefined
-      && checklist.geofenceBottom !== undefined
-      && checklist.geofenceLeft !== undefined
-      && checklist.geofenceRight !== undefined;
+    const unlockStartMissionButton : boolean = information[missionName].parameters !== undefined
```

Comment:
you actually dont need to declare the boolean type here, should be implied already